app/src/main/res/layout/activity_main.xml

```diff
@@ -16,87 +16,84 @@
         android:text="@string/drag_me"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:id="@+id/first"
         style="@style/AppTheme.CircleTextView"
-        android:layout_width="@dimen/circle_size"
-        android:layout_height="@dimen/circle_size"
+        android:layout_width="@dimen/other_circles_size"
+        android:layout_height="@dimen/other_circles_size"
         android:layout_marginTop="@dimen/circle_distance"
         android:backgroundTint="#388e3c"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/drag"/>
+        app:layout_constraintTop_toBottomOf="@+id/drag" />
 
     <TextView
         android:id="@+id/second"
         style="@style/AppTheme.CircleTextView"
-        android:layout_width="@dimen/circle_size"
-        android:layout_height="@dimen/circle_size"
+        android:layout_width="@dimen/other_circles_size"
+        android:layout_height="@dimen/other_circles_size"
         android:layout_marginTop="@dimen/circle_distance"
         android:backgroundTint="#81c784"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/first"/>
+        app:layout_constraintTop_toBottomOf="@+id/first" />
 
     <TextView
         android:id="@+id/damping_label"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:paddingEnd="8dp"
+        android:paddingStart="8dp"
         android:text="@string/damping_ratio"
-        android:layout_marginStart="12dp"
-        app:layout_constraintBottom_toTopOf="@+id/damping_value"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="1"
-        app:layout_constraintVertical_chainStyle="packed"/>
+        app:layout_constraintBottom_toTopOf="@id/damping_value" />
```

Comment:
This needs to be `@+id/damping_value` since the `damping_value` id has not been created at this point